PA193_mnemonic_Slytherin/mnemonic.py

```diff
@@ -38,24 +40,102 @@ def __generate_seed(mnemonic: str, seed_password: str = '') -> bytes:
 # - dictionary is too big (2048 lines OK, but too long words) like hundreds of MB...
 # - every line has exactly 1 word (no whitespaces)
 
-def __entropy2mnemonic(entropy: bytes) -> str:
-    """Convert entropy to mnemonic phrase using dictionary.
+def __get_dictionary() -> Tuple[List[str], Dict[str, int]]:
+    """Load the dictionary.
     Currently uses 1 default dictionary with English words.
     # TODO Should we support multiple dictionaries for various languages?
+    :raises FileNotFoundError: on missing file
+    :raises ValueError: on invalid dictionary
+    :rtype: Tuple[List[str], Dict[str, int]]
+    :return: List and dictionary of words
+
+    """
+    l = []
+    with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'english.txt'), 'r') as f:
```

Comment:
Please place filename to parameter with default value so we could write tests for various *malicious* dictionaries.
~~~py
ENGLISH_DICTIONARY_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'english.txt')


def __get_dictionary(file_path: str=ENGLISH_DICTIONARY_PATH) -> Tuple[List[str], Dict[str, int]]:
~~~